code/modules/mob/mob-logout.dm

```diff
@@ -34,13 +34,8 @@
 	active_storage?.hide(src)
 	update_client_z(null)
 	log_access_out(src)
-	if(admin_datums[src.ckey])
-		if (SSticker && SSticker.current_state == GAME_STATE_PLAYING) //Only report this stuff if we are currently playing.
-			var/admins_number = GLOB.admins.len
 
-			message_admins("Admin logout: [key_name(src)]")
-			if(admins_number == 0) //Apparently the admin logging out is no longer an admin at this point, so we have to check this towards 0 and not towards 1. Awell.
-				send2irc("LOGOUT", "[key_name(src)] logged out - no more admins online.")
+	// admin logout shenanigans moved to client destroy!
```

Comment:
dont need to comment it, anyone editing this file should know what they're doing enough to look